src/modules/launcher/Plugins/Microsoft.Plugin.WindowWalker/Components/NativeMethods.cs

```diff
@@ -847,10 +852,10 @@ public enum ExtendedWindowStyles : uint
         [DllImport("user32.dll", SetLastError = true)]
         public static extern uint GetWindowThreadProcessId(IntPtr hWnd, out uint lpdwProcessId);
 
-        [DllImport("psapi.dll")]
+        [DllImport("psapi.dll", BestFitMapping = false)]
         public static extern uint GetProcessImageFileName(IntPtr hProcess, [Out] StringBuilder lpImageFileName, [In][MarshalAs(UnmanagedType.U4)] int nSize);
 
-        [DllImport("user32.dll", SetLastError = true)]
+        [DllImport("user32.dll", SetLastError = true, BestFitMapping = false)]
```

Comment:
(Clarification) What unicode characters would get converted to ANSI?  When would we use `ThrowOnUnmappableChar` and when would we want to set `BestFitMapping` to true?

Reply:
@alekhyareddy28 In these P/inoke calls the parameter is an LPTSTR, so unicode would be unsuitable.   I chose BestFitMapping to fix the security holes.  It might be preferable to directly call the Unicode equivalents such as GetClassNameW directly, but I would like to fix that as a separate issue, If we see '?' characters in the window names.  I don't want to throw here as that would change the functionality and my intent was to enable fxcop and fix any warnings/errors.